src/status_im/subs/wallet/activities.cljs

```diff
@@ -85,29 +84,61 @@
            :token-id         token-id)))
 
 (defn- process-activity
-  [{:keys [activity-type token-out amount-out token-in amount-in approval-spender] :as activity} context]
+  [{:keys [activity-type symbol-out token-out amount-out symbol-in token-in amount-in approval-spender]
+    :as   activity}
+   context]
   (let [base-activity (process-base-activity activity context)]
     (condp = activity-type
       activity-constants/wallet-activity-type-send
       (assoc base-activity
              :tx-type    :send
-             :amount-out (get-token-amount token-out amount-out))
+             :amount-out (get-token-amount
+                          (assoc token-out
+                                 :token-symbol symbol-out
+                                 :decimals     (get-in context
+                                                       [:tokens-by-symbol symbol-out
+                                                        :decimals])
+                                 :amount       amount-out)))
 
       activity-constants/wallet-activity-type-bridge
       (assoc base-activity
              :tx-type    :bridge
-             :amount-out (get-token-amount token-out amount-out))
+             :amount-out (get-token-amount
+                          (assoc token-out
```

Comment:
This `assoc` code repeats 5 times in conjunction with `(get-token-amount)`, so we can move it inside the function. wdyt?

Reply:
Yes, we need to prevent the DRY code 🙏 . I will clean up 👍